### ISSUE TITLE
Question about training data

### ISSUE THREAD

**Issue by MooreManor** [ArtLabss/tennis-tracking#31]
> To predict bounce points machine learning library for time series [sktime](https://www.sktime.org/en/stable/index.html) was used. Specifically, [TimeSeriesForestClassifier](https://github.com/ArtLabss/tennis-tracking/blob/90652b4547311423ea49c4195dde9da9a81f1893/clf.pkl) was trained on 3 variables: x, y coordinates of the ball and V for velocity (V2-V1/t2-t1). Data for training the model - [df.csv](https://github.com/ArtLabss/tennis-tracking/blob/main/bigDF.csv)

Hello! @shukkkur 

I have a few questions about the training data of the bounce model. 

- According to the above content, you gave a label csv for the training data of the bounce model. What's the corresponding videos or images of that? Can you share the data?
- I want to retrain the bounce model, as [this issue](https://github.com/ArtLabss/tennis-tracking/issues/18#issuecomment-1210177886)  offered the training script, but the link is out of date now. Can you upload the script again?
- For training the tennis trajectory model (I.e. TrackNet), have you tried training it before? Do you have the training data of the original TrackNet?

Thanks!

**Comment by nyck33** [comment]
Same here, just asked ChatGPT with Noteable plugin and it led nowhere: https://chat.openai.com/share/5829196a-05cb-4b84-abef-9cad16bca6b0

**Comment by MooreManor** [comment]
@nyck33 

https://github.com/ArtLabss/tennis-tracking/issues/33#issuecomment-1607304448

It seems that the training script has lost.

**Comment by RohaanA** [comment]
> Same here, just asked ChatGPT with Noteable plugin and it led nowhere: https://chat.openai.com/share/5829196a-05cb-4b84-abef-9cad16bca6b0

Hi, Did you have any success in improving the bounce detection?
